### DISCUSSION TITLE
Sand for sandbox

### DISCUSSION THREAD

**Discussion by SBorderie** [cgre-aachen/open_AR_Sandbox#69] [Q&A]
Hello,
We just built an augmented reality sandbox in Fribourg, Switzerland. We combined AR Sandbox with GemPy and it works perfectly. I just have a problem with the sand. I bought kinetic sand from https://ar-sandbox.eu/product/kinetic-sand/ and the quality is really bad. The sand is very cohesive and very difficult to handle. So I would like some advice on what sand to use and where to get it.
Thanks !

**Comment by Japhiolite** [comment]
@SimonVirgo may be able to provide some suggestions

**Comment by SimonVirgo** [comment]
Hello,
The best way I found to finetune the parameters of kinetic sand is to mix it with fine grand pure Quarz sand and silicone oil. 
the sand should ideally not include any clay minerals and must be very, very dry before you start mixing. 
for the silicone oil, if you aim on reducing cohesion/ plasticity try a low viscosity silicone oil (1-10 cSt)
Start experimenting in small batches and scale up when you find a rheology you like. 

Hope that helps
Simon
